packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-dr.test.ts

```diff
@@ -159,8 +226,15 @@ describe("dr-response", () => {
           outboundRequest: outboundDrRequest,
         },
       });
+      const issueIds = response.operationOutcome?.issue.map(issue => issue.id);
+      const detailIds = response.operationOutcome?.issue.map(issue => issue.details?.id);
 
       expect(response?.operationOutcome?.issue[0]?.code).toEqual(schemaErrorCode);
+
+      expect(issueIds).toContain(outboundDrRequest?.documentReference?.[0]?.metriportId);
```

Comment:
would be good to define a const for `outboundDrRequest?.documentReference?`, make it expected that it exists, then reuse for the following statements